docs/jwst.rst

```diff
@@ -0,0 +1,254 @@
+*****************************
+
+
+The following describes specific details for the various JWST instrument classes. See also :ref:`the references page <references>` for information on data sources. 
+
+
+One general note is that the webbpsf class interfaces do not attempt to exactly
+model the implementation details of all instrument mechanisms, particularly for
+NIRCam and NIRISS that each have multiple wheels. In webbpsf, the
+``filter`` attribute of a given class is used to select any and all filters,
+even if as a practical matter a given filter is physically installed in a
+"pupil" wheel instead of a "filter" wheel. Likewise any masks that affect the
+aperture shape are selected via the ``pupil_mask`` attribute even if physically
+an optic is in a so-called "filter" wheel.
+
+
+Each SI has a ``detector`` attribute that can be used to select among its
+multiple detectors (if more than one are present in that SI), and a
+``detector_position`` attribute which is a tuple givin the pixel coordinates on
+that detector. Note that the ``detector_position`` value should be specified
+using the Python (Y,X) axes order convention.
+
+.. admonition:: Instrument measured WFE coming in a future release
+
+        This version of webbpsf does not yet contain data on the measured SI
+        wavefront errors. However it does contain software infrastructure to
+        support that functionality in the near future, once the data files have
+        been fully reviewed by the project and SI teams. So for the time being,
+        some options related to detector field positions and SI wavefront error
+        are present in webbpsf, but do not have any effect on the output PSFs.
+        Stay tuned!
+
+
+.. warning::
+    webbpsf provides some sanity checking on user inputs, but does not strive to
+    strictly forbid users from trying to simulate instrument configurations that 
+    may not be achievable in practice.  Users are responsible for knowing the
+    available modes well enough to be aware if they are trying to
+    simulate an inconsistent or physically impossible configuration. 
+
+
+NIRCam
+======
+
+Imaging
+--------
+
+NIRCam is one of the more complicated classes in webbpsf, and has several unique selectable options to model the two copies of NIRCam each with two channels..  
+
+The ``detector`` attribute can be used to select between any of the ten detectors,
+A1-A5 and B1-B5.  Additional attributes are then automatically set for ``channel``
+("short" or "long") and module ("A" or "B") but these cannot be set directly;
+just set the desired detector and the channel and module are inferred
+automatically. 
+
+The choice of ``filter`` also impacts the channel selection: If you choose a
+long-wavelength filter such as F460M, then the detector will automatically
+switch to the long-wave detector for the current channel. For example, if the
+detector was previously set to A2, and the user enters ``nircam.filter="F460M"``
```

Comment:
Nitpick: spaces on either side of the `=` are PEP8 style (even in a tiny code snippet)